Example/.swiftlint.yml

```diff
@@ -14,7 +14,7 @@ excluded:
 - Pods
 
 line_length:
-    warning: 150
```

Comment:
Usually we have 120. I know that this is hard for urls but therefore we disable it for the line explicitly.

_I would prefer to change to 120 (or 150). But 200 is to high in my opinion_

Reply:
changed to 120